pkg/runtime/runtime.go

```diff
@@ -2472,8 +2472,18 @@ func (a *DaprRuntime) loadComponents(opts *runtimeOpts) error {
 	a.components = authorizedComps
```

Comment:
With the change below should a.components also be changed to maintain the same order?

Reply:
I looked into that but personally don't think it's needed. Do you think it would make a difference?

Reply:
looked into the code and I think you are right, the order in the components slice does not seem to make a difference.